main.py

```diff
@@ -1,10 +1,40 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as json_file:
+        data = json.load(json_file)
+    for nickname, user in data.items():
+        race_info = user["race"]
+        race, created = Race.objects.get_or_create(
```

Comment:
You could omit the `created` variable with `_` since it is not used in the logic.